### DISCUSSION TITLE
Can I configure "default settings" for every new Sandbox?

### DISCUSSION THREAD

**Discussion by stdedos** [sandboxie-plus/Sandboxie#3731] [Documents and notes]
e.g, I would like to always block "System Protected Storage" and "Windows Credential Store"

![image](https://github.com/sandboxie-plus/Sandboxie/assets/133706/5c276b04-85b7-4dc5-8d77-deb20efa6619)

_idk if it's a good idea, but I'd "risk" to find out_

**Comment by offhub** [comment]
Isn't that what `[GlobalSettings]` is for?

**Reply by stdedos** [reply]
I guess yes! Now I found the relevant section I was interested:
https://sandboxie-plus.com/sandboxie/sandboxieini/

> Global Settings:
> ...
> Sandbox settings may appear in the GlobalSettings section, and can be overridden by also including them in a sandbox section.

Sorry / Thank you 🙃

**Reply by bastik-1001** [reply]
Additionally, the box creation wizard (part of Sandman) allows you to save new default settings for boxes you create with it.

`Sandbox > Create New Box` opens the wizard. In it, you choose your box-type and if you check "Configure advanced options" you can set up the box with different settings and when you made it to the final page where the button changes from "Next" to "Finish" there is another checkbox "Save option as new default."

**Reply by stdedos** [reply]
So I added:

```ini
[GlobalSettings]

Template=RpcPortBindings
Template=7zipShellEx
Template=WindowsRasMan
Template=OpenProtectedStorage
```

Am I supposed to see something different here now?
![image](https://github.com/sandboxie-plus/Sandboxie/assets/133706/2ffbccdc-b839-440c-8946-5700bb850944)

**Reply by bastik-1001** [reply]
Not all settings, when enabled globally, are reflected in the box settings. I found [https://github.com/sandboxie-plus/Sandboxie/issues/3193](https://github.com/sandboxie-plus/Sandboxie/issues/3193) ImageProtection to be one.
